### DISCUSSION TITLE
Save as Pan 0

### DISCUSSION THREAD

**Discussion by mdaconta** [AxisCommunications/discussions#96] [General] [answered]
Hi Axis folks,

We are integrating with the Axis Q62 PTZ Camera, and in your web interface there is a setting to "Save as Pan 0" (degrees).  Can you tell me the API that function uses? ... if possible, I want to implement it via the Onvif interface.

Thanks for any information on this,

-Mike

**Comment by mattias-kindborg-at-work** [comment, marked as answer]
Hi @mdaconta. I don't think that there's support for setting this parameter using Onvif, but using VAPIX you can set the parameter (`root.PTZ.Various.V1.PanOffset`) using `param.cgi`. For more information regarding parameter management, please see the [VAPIX documentation](https://www.axis.com/vapix-library/subjects/t10175981/section/t10036014/display).

**Reply by mdaconta** [reply]
Thanks Mattias!  I appreciate the answer and will examine that documentation.
Regards,
-Mike
